packages/omg-js-childchain/src/transaction/transaction.js

```diff
@@ -14,14 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License. */
 
 const { InvalidArgumentError } = require('@omisego/omg-js-util')
+const Web3Utils = require('web3-utils')
 
 const MAX_INPUTS = 2
 const MAX_OUTPUTS = 2
 
 const NULL_INPUT = { blknum: 0, txindex: 0, oindex: 0 }
 const NULL_OUTPUT = { owner: '0x0000000000000000000000000000000000000000', amount: 0 }
 
-const BLOCK_OFFSET = 1000000000
+const BLOCK_OFFSET = Web3Utils.toBN(1000000000)
```

Comment:
What was the issue here? Related to #31?

Reply:
Yeah, Javascript can't handle numbers over 9007199254740991 (2^53), whereas ethereum integers are 2^256
The issue is that if the blocknum is large (> 9000000) then we'll hit the limitation and the resulting utxoPos will be wrong.
In general we need to use a BigNumber class when doing arithmetic in the lib, issue #31 refers to choosing between BN.js or BigNumber.js